inc/functions/files.php

```diff
@@ -613,10 +613,13 @@ function rocket_clean_home( $lang = '' ) {
 	do_action( 'before_rocket_clean_home', $root, $lang ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
 
 	// Delete homepage.
-	$files = glob( $root . '/{index,index-*}.{html,html_gzip}', GLOB_BRACE | GLOB_NOSORT );
```

Comment:
Will this delete also other files and folders? I did not tested it, but I am concerned it might delete other files except index files.

Reply:
The new regex should not delete any other files. @piotrbak has given it some initial tests (and found in the process this separate issue #3444) and confirmed that it should be ok, but has not had full QA yet.
Some discussion came up last week re: glob use in general. See my comment [here](https://github.com/wp-media/wp-rocket/issues/146#issuecomment-753994031) 
@crystinutzaa tagging you for Code Review.